### ISSUE TITLE
Field status highlights look a bit broken in SuperTable blocks using Row Layout

### ISSUE THREAD

**Issue by mmikkel** [engram-design/SuperTable#433]
**Description**

The styling for the field status highlights (aka field modification badges, aka those vertical blue lines) look a bit off for nested SuperTable fields when using the "Row Layout" field layout.

This particular ST field is nested in a Matrix block; not sure if the issue applies to standalone SuperTable fields as well.

![image](https://user-images.githubusercontent.com/298510/136197579-58d635af-7f41-44d0-a25e-0a053bdeee6a.png)

**Steps to reproduce**
1. Create a SuperTable field in a Matrix field, select the "Row Layout" field layout and a couple of nested fields
2. Make some changes to the nested fields to make the field status highlights appear

**Additional info**
- Plugin version: 2.6.8
- Craft version: 3.7.15
- Multi-site: No

**Additional context**
The above screenshot is from Edge 94 on macOS Big Sur 11.6. I haven't tested on any other browsers.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-3 as 2.6.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [2.7.0](https://github.com/verbb/super-table/releases/tag/2.7.0)
